.gitignore

```diff
@@ -1,4 +1,5 @@
 frontend/package-lock.json
+env/
```

Comment:
Yeah this should do it. This should keep git from recognizing any changes in the `env/` directory. Now you just need to delete those files. I think you might have made a `venv/` directory as well? If so, like if you still have that, you'll need to add it too. 

And then you also need to delete everything that isn't the `.gitignore` file or the frontend changes you made. 

Do you have a `venv/` directory too on your remote? If so, then also add it here. And then you still need to delete all the changes you made ALL OF THEM, except this `.gitignore` file and the frontend changes you made and want to submit.